libs/base/Data/List1/Quantifiers.idr

```diff
@@ -23,6 +23,11 @@ namespace Any
     ||| A proof that the satisfying element is in the tail of the `List1`
     There : {0 xs : List a} -> Any p xs -> Any p (x ::: xs)
 
+  export
+  {0 p : a -> Type} -> Uninhabited (p x) => Uninhabited (Any p xs) => Uninhabited (Any p $ x:::xs) where
+    uninhabited (Here y) = uninhabited y
+    uninhabited (There y) = uninhabited y
+
```

Comment:
Could this not be done via `All (Uninhabited . p) xs => Unihabited (Any p xs)`?

Reply:
This definitely could be. I just copied this way from `List.Quantifier`. I would agree that the way using `All` seems to be slighly more universal, but then we need to change it in all modules (I'm ready to do this).